nri-config-generator/integration-tests/generator_test.go

```diff
@@ -22,6 +21,7 @@ import (
 const (
 	testIntegrationVersion = "test-tag"
 	testIntegration        = "nri-powerdns"
```

Comment:
could we remove these then?

Reply:
that is still used to run the config generator. Templates are copied to a local folder used by the generator.